examples/basic_sg_distributed_index_config.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
I don't think we expect anyone external to know what 'distributed index' means - a better name might be something like 'sync_gateway_with_accel_config.json'